.circleci/config.yml

```diff
@@ -11,13 +11,27 @@ parameters:
     type: string
     default: dps-releases
 
+executors:
+  validator:
+    docker:
+      - image: cimg/openjdk:16.0-browsers
```

Comment:
since this is new you might want to try Java 17 image, DPS Tech Team are trialling this as we speak

Reply:
yes - was thinking that for the next PR but might as well do it now